pkg/kube/snapshot/snapshot_beta.go

```diff
@@ -63,7 +63,7 @@ func cloneSnapshotClass(ctx context.Context, dynCli dynamic.Interface, snapClass
 	for _, key := range excludeAnnotations {
 		delete(existingAnnotations, key)
 	}
-	usNew := UnstructuredVolumeSnapshotClass(snapClassGVR, targetClassName, sourceSnapClass.Driver, newDeletionPolicy)
+	usNew := UnstructuredVolumeSnapshotClass(snapClassGVR, targetClassName, sourceSnapClass.Driver, newDeletionPolicy, sourceSnapClass.Parameters)
```

Comment:
Hi @bathina2,
By introducing this argument it looks like we are assuming `params` field will always be there in the `VolumeSnapshotClass`. But I have seen some examples where params field is not present in the `VoluemSnapshotClass` resoruce, and because of that when we try to create the copied `VolumeSnapshotClass` with `nil` as parameter value VolumeSnapshotClass Creation fails with below error 
```
"k10-clone-csi-hostpath-snapclass" is invalid: parameters: Invalid value: "null": parameters in body must be of type object: "null"
```
I was thinking we shouldn't pass the parameters field if the values is nil.